WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -177,25 +177,35 @@ private extension PersonViewController {
         alert.addCancelActionWithTitle(cancelTitle)
 
         alert.addDestructiveActionWithTitle(removeTitle) { action in
-            self.removePerson()
+            self.deleteUser()
         }
 
         alert.presentFromRootViewController()
     }
 
-    func removePerson() {
+    func deleteUser() {
+        guard let user = user else {
+            DDLogSwift.logError("Error: Only Users can be deleted")
+            return
+        }
+
         let service = PeopleService(blog: blog)
-        service?.deletePerson(person)
+        service?.deleteUser(user)
         navigationController?.popViewControllerAnimated(true)
     }
 
-    func updateRole(newRole: Person.Role) {
+    func updateUserRole(newRole: Role) {
+        guard let user = user else {
+            DDLogSwift.logError("Error: Only Users have Roles!")
```

Comment:
Same as above... if we're logging an error it means this should be an assertion failure in debug builds.